### PR TITLE
testing/{openttd, openttd-gfx, openttd-sfx}: new aports

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

File: testing/openttd/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+package() {
+	cd "$builddir"
+	make install || return 1
+}
+
+lang() {
+	cd "$pkgdir"
+	mkdir -p "$subpkgdir"/usr/share/games/openttd/lang
+	for i in $(find usr/share/games/openttd/lang/ ! -name 'english.lng' -type f); do
```
Comment (jirutka) [inline testing/openttd/APKBUILD]: Variables used only in a single function should be declared as `local` (i.e. `local i; for i in …`).

File: testing/openttd/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$builddir"
+	make install || return 1
+}
+
+lang() {
```
Comment (jirutka) [inline testing/openttd/APKBUILD]: Please declare variable `pkgdesc` inside this function with description of the subpackage (e.g. `pkgdesc="$pkgdesc (localizations)"`).

File: testing/openttd/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+pkgname=openttd
+pkgver=1.6.1
+pkgrel=0
+pkgdesc="Open source simulation game based upon Transport Tycoon Deluxe"
+url="http://openttd.org/"
+arch="all"
+license="GPL2"
+depends=""
+makedepends="libpng-dev zlib-dev sdl-dev lzo-dev xz-dev freetype-dev fontconfig-dev"
+subpackages="$pkgname-doc $pkgname-lang"
```
Comment (jirutka) [inline testing/openttd/APKBUILD]: Declare -lang subpackage as noarch (i.e. `$pkgname-lang::noarch`).

File: testing/openttd/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+	# but it will be done automatically later in the package build.
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--disable-strip \
+		--prefix-dir=/usr \
+		--binary-dir=bin \
+		--install-dir="${pkgdir}" \
+		--with-sdl --with-zlib --with-liblzma --with-liblzo2 \
+		--with-freetype --with-fontconfig --without-icu \
```
Comment (jirutka) [inline testing/openttd/APKBUILD]: Please keep each option on single line.

File: testing/openttd/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# Contributor: Adrian Siekierka <kontakt@asie.pl>
+# Maintainer: Adrian Siekierka <kontakt@asie.pl>
+pkgname=openttd
+pkgver=1.6.1
+pkgrel=0
+pkgdesc="Open source simulation game based upon Transport Tycoon Deluxe"
+url="http://openttd.org/"
+arch="all"
+license="GPL2"
+depends=""
```
Comment (jirutka) [inline testing/openttd/APKBUILD]: I think that you’ve reversed dependencies – openttd should depend on `openttd-opensfx` and `openttd-opengfx`, not vice versa.

Comment (asiekierka) [inline testing/openttd/APKBUILD]: Actually, this is correct - OpenGFX and OpenSFX are not strictly required for OpenTTD to run and, if they're not installed globally, OpenTTD will ask if you want to download and install them locally. You can also use the original assets from Transport Tycoon Deluxe, or a different project called zBase.

File: testing/openttd/APKBUILD (hunk 2 above)
Comment (asiekierka) [inline testing/openttd/APKBUILD]: Resolved

**Pull request closed without merging**